Assets/Plugins/Android/Core/AndroidFileIOHelper.cs

```diff
@@ -23,23 +23,43 @@
 namespace PlayEveryWare.EpicOnlineServices
 {
     using System;
+    using System.IO;
     using System.Threading.Tasks;
     using UnityEngine;
     using UnityEngine.Networking;
 
     public class AndroidFileIOHelper
     {
-        public static async Task<string> ReadAllText(string filePath)
+        public static bool FileExists(string filePath)
         {
             using UnityWebRequest request = UnityWebRequest.Get(filePath);
-            request.timeout = 2; //seconds till timeout
-            UnityWebRequestAsyncOperation operation = request.SendWebRequest();
+            request.timeout = 2;
+            request.SendWebRequest();
+            while (!request.isDone) { }
```

Comment:
I was pretty uncertain about this, but it appears as though this works properly. I made this testing class to validate the behaviour, and it correctly does finish. The app appears to be locked up until it's done, but that's the intentional and desired synchronous action.

```csharp
    private void Update()
    {
        if (Input.GetKeyDown(KeyCode.A))
        {
            BeginTest();
        }
    }

    private void BeginTest()
    {
        StatusText.text = "Beginning Test";

        using UnityWebRequest request = UnityWebRequest.Get(@"REALLY HUGE ZIP FILE");
        request.timeout = 2;
        request.SendWebRequest();

        StatusText.text = "Waiting for test to finish";

        while (!request.isDone) { }

        StatusText.text = $"Test finished {request.result}";
    }
```